ledger/src/ancestor_iterator.rs

```diff
@@ -50,6 +50,28 @@ impl<'a> Iterator for AncestorIterator<'a> {
     }
 }
 
+pub struct AncestorIteratorWithHash<'a> {
```

Comment:
Is this going to be used later?

Reply:
yup!

Reply:
no unit test? :trollface:

Reply:
good call! added here: https://github.com/solana-labs/solana/pull/18533/commits/c6aa6516f4e68143eeff7b7610c69fee3375a99c